Assets/Mirror/Runtime/NetworkManager.cs

```diff
@@ -390,9 +390,9 @@ public void StartClient(Uri uri)
         public virtual void StartHost()
         {
             OnStartHost();
-            NetworkClient.SetupLocalConnection();
```

Comment:
There is a subtle bug here.

I moved SetupLocalConnection before StartServer because OnServerStart has to know about the local client or it won't fire hooks.  Basically you are reintroducing the problem fixed in #1249

Reply:
that would've been worth a comment there :)

Reply:
adding...